compute/containers/api-cli/restricting-access-to-a-container.mdx

```diff
@@ -0,0 +1,100 @@
+
+</Message>
+
+
+<Message type="requirement">
+
+  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
+  - You have generated a Scaleway [API key](https://www.scaleway.com/en/docs/console/my-project/how-to/generate-api-key/)
+  - You have either `curl`, [Postman](https://www.postman.com/), [Talend API tester](https://chrome.google.com/webstore/detail/talend-api-tester-free-ed/aejoelaoggembcahagimdiliamlcdmfm) or any tool to send HTTP requests
+
+</Message>
+
+## Generating a JWT token
+
+Privacy can be ensured via **J**SON **W**eb **T**okens. A JWT Token can be retrieved from the API-endpoint `GET /issue-jwt`. Depending on the parameters, a JWT token can be valid for either a function, a container, or a namespace:
+
+* `/issue-jwt?namespace_id=1`: issues JWT valid for all functions inside namespace with ID 1.
+* `/issue-jwt?function_id=1`: issues JWT valid only for function with ID 1.
+
+<Message type="tip">
+
+You may optionally provide an expiration date (formatted `yyyy-mm-ddT00:00:00Z`) for the token. 
+For example: `/issue-jwt?expires_at=2022-01-02T00:00:00Z&namespace_id=1` will generate a token, valid for all functions and containers inside namespace with ID 1, and this token will be valid until January 2nd 2022.
+
+</Message>
+
+The token will have the following claims:
+
+```yaml
+{
+ "application_claim": [
+   {
+     "namespace_id": "string",
+     "container_id": "string" // optional: id of the container
+   }
+ ]
+}
+```
+
+<Message type="important">
+
+* Tokens are not stored by Scaleway and can not be retrieved if lost. However, new tokens can be generated.
+* Token revocation is not yet supported. The best way to reset tokens is to destroy and recreate the namespaces and all of their functions.
+
+</Message>
+
+### Using curl
+
+Use the following command to issue a JWT token for your namespace from the command line using cURL: 
+
+```sh
+curl --location --request GET 'https://api.scaleway.com/functions/v1beta1/regions/fr-par/issue-jwt?namespace_id={your_namespace_id}&expires_at=2022-06-22T00%3A00%3A00Z' --header 'X-Auth-Token: {your_scaleway_API_key'}
+```
+
+### Using an API Tester (Talend API Tester)
+
+The following steps are designed for [Talend API tester](https://chrome.google.com/webstore/detail/talend-api-tester-free-ed/aejoelaoggembcahagimdiliamlcdmfm). 
+
+1. Specify the `GET` method.
+2. Enter the API URL. 
+    * For namespaces:
+      ```
+      https://api.scaleway.com/functions/v1beta1/regions/fr-par/issue-jwt?namespace_id={your_namespace_id}&expires_at={expiration_date}
+      ```
+    * For Containers:
+      ```
+      https://api.scaleway.com/functions/v1beta1/regions/fr-par/issue-jwt?container_id={your_container_id}&expires_at={expiration_date}
+      ```
+3. Add a `X-Auth-Token` header with your Scaleway API key as value.
+4. Send the request.
+5. Copy the returned token.
+
+<Lightbox src="scaleway-talend-container.png" alt="" />
+
+## Using your token
```

Comment:
Perhaps "Validate tokens inside your container" is better as using your token might refer to the "client side"